api/graphql/schema.graphql

```diff
@@ -730,6 +730,9 @@ input UpdateUserAsUserInput {
     communityExperiences: CommunityExperienceHasMany
     educationExperiences: EducationExperienceHasMany
     awardExperiences: AwardExperienceHasMany
+
+    # Profile Status
+    isProfileComplete: Boolean
```

Comment:
> However, isProfileComplete is already part of the User and Applicant types in the schema, from a previous PR. The place where you've added isProfileComplete to the schema in this PR is an `input` type, which defines what you can change in a mutation. This doesn't work, because isProfileComplete can't be set in a mutation.

Actually isProfileComplete is not added to Applicant types yet. Do we need that? I was thinking same but without adding it is giving typscript error when I added to mutations. I will try again and let you know.